analytics-api/migrations/versions/05e82fada6fe_add_runcycle_id_to_all_tables.py

```diff
@@ -0,0 +1,56 @@
+"""add_runcycle_id_to_all_tables
+
+Revision ID: 05e82fada6fe
+Revises: e62ba34dc486
+Create Date: 2022-10-06 11:54:03.453343
+
+"""
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = '05e82fada6fe'
+down_revision = 'e62ba34dc486'
+branch_labels = None
```

Comment:
i hope these revision numbers are correct and the scripts wont run when we deploy

Reply:
Yes its the same we had within met-cron.